MekHQ/src/mekhq/gui/view/AtBScenarioViewPanel.java

```diff
@@ -771,68 +768,13 @@ protected Icon getIcon(Object node) {
         }
 
         protected Icon getIconFrom(UnitStub unit) {
```

Comment:
So many copies of this block of code! (including the comment with the false claim of returning `null`)

```
$ grep -r '// Return a null if the player has selected no portrait file.' src/
src//mekhq/gui/view/ForceViewPanel.java:        // Return a null if the player has selected no portrait file.
src//mekhq/gui/view/ForceViewPanel.java:        // Return a null if the player has selected no portrait file.
src//mekhq/gui/view/ScenarioViewPanel.java:            // Return a null if the player has selected no portrait file.
src//mekhq/gui/view/ScenarioViewPanel.java:            // Return a null if the player has selected no portrait file.
src//mekhq/gui/view/AtBScenarioViewPanel.java:            // Return a null if the player has selected no portrait file.
src//mekhq/gui/view/AtBScenarioViewPanel.java:            // Return a null if the player has selected no portrait file.
src//mekhq/gui/view/PersonViewPanel.java:        // Return a null if the player has selected no portrait file.
src//mekhq/gui/BasicInfo.java:            // Return a null if the player has selected no portrait file.
src//mekhq/gui/BasicInfo.java:            // Return a null if the player has selected no portrait file.
```